javalin/src/main/java/io/javalin/core/JavalinConfig.java

```diff
@@ -104,37 +103,42 @@ public JavalinConfig enableDevLogging() {
     }
 
     public JavalinConfig enableWebjars() {
-        return addStaticFiles("/webjars", Location.CLASSPATH);
-    }
-
-    public JavalinConfig addStaticFiles(@NotNull String classpathPath) {
```

Comment:
It might be useful to provide `Location.CLASSPATH` as default value for location in `addStaticFiles` method:

```java
public JavalinConfig addStaticFiles(@NotNull String directory) {
  return addStaticFiles(directory, Location.CLASSPATH);
}
```

It's used this way in various examples and tests, so we can just simplify these calls. It's also the most common usage in real-world apps.

Reply:
I removed the single-param-method because I've found myself writing comments on how the lack of a `Location` means `Location.CLASSPATH` (both in the tests and in my own apps).

Requiring the location also signals to developers that it is in fact possible to change the location to something other than `Location.CLASSPATH`.

Reply:
You can always name it just like e.g. `addClasspathStaticFiles` 😅 But I see, just preferred to ask to be sure.

Reply:
I did consider something like `staticFiles.addClasspath` and  `staticFiles.addExternal` 🤔

We can always add that later if there is a demand for it, but it's impossible to remove something after it's out, so I prefer to be a bit restrictive.